File: src/Belajar_SOLID_04_OOP_Abstraction/Java/Main.java
package Belajar_SOLID_04_OOP_Abstraction.Java;

public class Main {
    public static void main(String[] args) {
        // Membuat object dari Triangle
        Shape segitiga = new Triangle(4,3);

        // Membuat object dari circle
        Shape lingkaran = new Circle(10);

        System.out.println("Luas Lingkaran : " + lingkaran.getArea() );
        System.out.println("Luas Segitiga : " + segitiga.getArea() );
    }
}

File: src/Belajar_SOLID_04_OOP_Abstraction/Kotlin/Main.kt
package Belajar_SOLID_04_OOP_Abstraction.Kotlin

fun main(){
    val lingkaran = Circle(20f)
    val segitiga = Triangle(5f,10f);

    println("Luas Segitiga ${segitiga.getArea()}")
    println("Luas Lingkaran ${lingkaran.getArea()}")
}